Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -108,6 +108,9 @@ def visit_ParallelAssignmentNode(self, node):
     def visit_CEnumDefNode(self, node):
         return self.visit_StatNode(node, True)
 
+    def visit_CppScopedEnumDefNode(self, node):
```

Comment:
These are extra changes that could potentially go away if you were able to use the existing `CEnumDefNode` or inheritance.

Reply:
Done! :)